client/components/dataviews/style.scss

```diff
@@ -153,6 +153,11 @@
 			}
 		}
 	}
+
+	// Color overrides for focus states of Action button
+	.components-button:focus:not(:disabled) {
+		box-shadow: 0 0 0 2px var(--color-primary-light);
```

Comment:
The original styles from Core:

<img width="894" alt="image" src="https://github.com/user-attachments/assets/a0dbd46a-c955-4dc0-b2a3-584f0f92db40">

I wonder whether the "more correct" approach here is to e.g. to redefine `--wp-admin-theme-color` to something else while in sites dashboard pages? (Not sure if that's possible at all)

Just a discussion, not something that needs to be fixed here 😄

Reply:
Sounds good! Let me try that out ☝️